docs/source/routing/observability/debugging-client-requests.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Debugging Client Requests to GraphOS Router
+subtitle: Log incoming requests to help debug problematic queries or errors
+description: Enable telemetry features to help gain more visibility into incoming HTTP requests and help debug issues your clients may be having.
+---
+
+GraphOS Router by default does not emit access logs for the incoming client HTTP requests, nor are other features, like OpenTelemetry, on by default that include every request. Given the volume and size of GraphQL requests, this would produce a large amount of telemetry data that may impact your observability costs.
```

Comment:
Wording is bit confusing here. Maybe rewrite to something like:
"By default, the GraphOS Router does not generate access logs for incoming client HTTP requests, nor are features like OpenTelemetry enabled to capture every request."